fragments/legacy-storage/ceph/bundle.yaml

```diff
@@ -0,0 +1,19 @@
+# This is an incomplete bundle fragment. Do not attempt to deploy.
```

Comment:
We should have a test for this (or at least a card to write a test) so as to make sure this works and we do not break it in the future.